bridge/src/main/java/io/stargate/bridge/service/ExceptionHandler.java

```diff
@@ -43,9 +43,9 @@ public class ExceptionHandler {
   static final Metadata.Key<QueryOuterClass.CasWriteUnknown> CAS_WRITE_UNKNOWN_KEY =
       ProtoUtils.keyForProto(QueryOuterClass.CasWriteUnknown.getDefaultInstance());
 
-  private final StreamObserver<QueryOuterClass.Response> responseObserver;
+  private final StreamObserver<?> responseObserver;
```

Comment:
Parameterization not used for anything. Could have changed it to something like `GeneratedMessageV3` (common base class for protobuf-generated message classes) but does not seem valuable.

Reply:
Leave it.. seems enough for exception handling